graypy/handler.py

```diff
@@ -183,6 +183,8 @@ def make_message_dict(record, debugging_fields, extra_fields, fqdn, localname,
     logging.INFO: 6,
     logging.DEBUG: 7,
 }
+if hasattr(logging, 'NOTICE'):
```

Comment:
Yes, the logging library in the stdlib doesn't have a `NOTICE` level. I think the `verboselogs` library monkey-patches it and adds some new levels.

This seems like an edge-case for me. I don't know if we should support it.